test/llilc_runtest.py

```diff
@@ -171,7 +173,8 @@ def main(argv):
     # Run the test
     return_code = const.RunTestOK
     exclusion = os.path.join(os.path.dirname(__file__), 'exclusion.targets')
-    runtest_command = 'runtest ' + args.arch + ' ' + args.build 
+    runtest_full_path = os.path.join(runtest_dir, 'runtest')
```

Comment:
The current approach won't work on Linux, which requires that we append the file extension (i.e. `.sh`) to the path. IMHO we should always test for the script that we're going to run.